### PR TITLE
Release 1.7.0

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

**Review by ylyangtw — APPROVED** [review]

File: run_update_usergroup.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python update_usergroup_to_db.py
```
Comment (longshuicy) [inline run_update_usergroup.sh:3]: is this file needed in this repo?

File: src/components/DFR3Viewer.jsx [hunk 2]
```diff
@@ -899,6 +896,9 @@ class DFR3Viewer extends React.Component {
 											<MenuItem value="hurricaneWindfield" className={classes.denseStyle}>
 												Hurricane Windfield
 											</MenuItem>
+											<MenuItem value="earthquake%2btsunami" className={classes.denseStyle}>
+												Earthquake + Tsunami
+											</MenuItem>
```
Comment (longshuicy) [inline src/components/DFR3Viewer.jsx:901]: @navarroc do we also want to hide this?

**Review by longshuicy — APPROVED** [review]
